samples/cpp/build_samples_msvc.ps1

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Arguments parsing
+param (
+    [string]$BuildDirectory = "",
+    [string]$InstallDirectory = "",
+    [switch]$Help,
+    [switch]$h
+)
+
+$SourceDirectory = Split-Path $MyInvocation.MyCommand.Path
+$SamplesType = (Get-Item $SourceDirectory).Name
+$BuildDirectory = if ($BuildDirectory) {$BuildDirectory} else {"$Env:USERPROFILE/Documents/Intel/OpenVINO/openvino_${SamplesType}_samples_build"}
+
+if ($Help -or $h) {
+    Write-Host "
+        Build OpenVINO Runtime samples
+        Options:
+            -Help/-h                Print the help message and exit
+            -BuildDirectory         Specify the samples build directory. Default is $BuildDirectory
+            -InstallDirectory       Specify the samples install directory
```

Comment:
Is it possible to align cmd args? They should be -b and -i. I'm OK, if you keep long version as alternative